### PR TITLE
Adding Library jquery.iOSList v2.0.6

### PR DESCRIPTION
**Pull request by abishekrsrikaanth**
Repo: https://github.com/brianhadaway/iOSList
URL: http://brianhadaway.github.io/iOSList
Description: The jQuery iOS (iPhone/iPad/iPod) Alphabetical Contact List

### CODE REVIEW

File: files/jquery.ioslist/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "jquery.ioslist",
+  "repo": "brianhadaway/iOSList",
+  "files": {
+	"basePath" : "dist/",
+    "include": ["js/jquery.ioslist.js", "css/jquery.ioslist.css"]
```
Comment (megawac) [inline files/jquery.ioslist/update.json]: Can you change these to the minified files

**Comment by megawac** [comment]
> Repo: https://github.com/brianhadaway/iOSList
URL: http://brianhadaway.github.io/iOSList

I love when people do that - thanks for making my life easier!

**Comment by abishekrsrikaanth** [comment]
@megawac I have added it as an additional commit, can you squash and merge. I am having trouble locally to squash. If it is trouble, I can close this and send a fresh PR.

**Comment by megawac** [comment]
No worries, thanks

@jsdelivrbot squish and squash

**Pull request closed without merging**